llumnix/entrypoints/vllm/api_server.py

```diff
@@ -250,7 +251,8 @@ def add_argument(self, *args, **kwargs):
     parser.add_argument('--disable-log-requests-server', action='store_true', help='disable logging requests in server')
     parser.add_argument("--ray-cluster-port", type=int)
     parser.add_argument('--launch-ray-cluster', action='store_true', help='if launch ray cluster in api server')
-    parser.add_argument("--request-output-queue-port", type=int)
+    parser.add_argument("--queue-type", type=str, choices=['rayqueue', 'zmq'], help='queue type for request output queue')
+    parser.add_argument("--request-output-queue-port", type=int, help='port for zeromq')
```

Comment:
We can test arguments in arguments checker.

Reply:
For arguments with specified choices, the argument checker will automatically validate the values.

Reply:
see llumnix/arg_utils.py::check_args